### ISSUE TITLE
Lexicon - keyboard bug after browser resizing

### ISSUE THREAD

**Issue by catbusofthecanals** [jadu/pulsar#1575]
**Describe the bug**
After being hidden at a smaller viewport or 200% zoom, when resizing from back to full desktop, the primary nav menu sidebar reappears but cannot be accessed again when tabbing by keyboard.

**To Reproduce**
1. Go to the Pulsar Lexicon
2. Resize to a smaller viewport (under 992 px in width) until the nav disappears
3. Resize again to desktop view (over 992 px in width)
4. Tab through page with keyboard

**Expected behavior**
When the primary nav menu is visible in desktop view, the menu items should be part of the focus order, after browserre-sizing.

**Screenshots**
<img width="1020" alt="Image" src="https://github.com/user-attachments/assets/9f6d80be-fc2b-4d5a-8306-df2844cbba18" />

**Additional context**
Related success criteria:
[2.1.1 Keyboard (A)](https://www.w3.org/WAI/WCAG22/Understanding/keyboard)

**Suggested fix**
Ensure when sidebar reappears at min-width: 992px, the nav menu list items are included in the tab order. This requires further investigation but it appears the manageTabIndexes function isn’t being properly called when the browser is resized, meaning the nav bar items remain with tabindex -1.
